src/AttachmentPreview/AttachmentPreview.less

```diff
@@ -0,0 +1,195 @@
+@import (reference) "../less/utilities";
+
+@HEADER_BAR_HEIGHT: @size_5xl;
+@DESKTOP_BUFFER_SIZE: @size_l;
+@MOBILE_BUFFER_SIZE: @size_xs;
+
+// TODO: can I add this to sizing.less?
```

Comment:
I think we'd need to check with Design first, as I think our pantheon of constants is kind of sacred.

Reply:
I'll just leave it here for now